sdk/storage/Azure.Storage.Files.DataLake/src/Models/PathHierarchyDeletedItem.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Files.DataLake.Models
+{
+    /// <summary>
+    /// Either a <see cref="Prefix"/> or <see cref="Path"/>.
+    /// </summary>
+    public class PathHierarchyDeletedItem
+    {
+        /// <summary>
+        /// Gets a prefix, relative to the delimiter used to get the paths.
+        /// </summary>
+        public string Prefix { get; internal set; }
```

Comment:
@kasobol-msft, since we are not exposing the delimiter parameter, I don't think it makes sense for us to include `Prefix` and `IsPrefix` on `PathHierarchyDeletedItem`, since they will always be null and false.  It might make sense to remove this class all together, and have `PathDeletedSegment` have an `IEnumerable<PathDeletedItem>` instead.

Reply:
makes sense.

Reply:
fixed.